### ISSUE TITLE
Understand Change in Behaviour of nuclei in g4 v4.11.1.p02 vs v4.10.7.p02e

### ISSUE THREAD

**Issue by kutschke** [Mu2e/Offline#1018]
During the validation of geant 4.11.1.p02 :

https://mu2ewiki.fnal.gov/wiki/GeantChecklist#Geant4_v4_11_1_p02

Ray pointed out one plot of interest https://mu2e.fnal.gov/atwork/computing/ops/val/plots/g4_11_1_p02/pot/Validation_SimParticlemv_NeutralsFilter_noName_pn2.gif

This is the momentum of nuclei that pass the neutrals filter.  Most of these nuclear are created in material far from any detector.  They decay to other nuclei and eventually to a photon or neutron that crosses the reporting plane.  These neutrons are about (0.6 +/- 0.1 )% of all neutrons that cross the reporting planes and they populate the lower end of the momentum spectrum.  These photons are about ( 8.6 +/- 0.4)% of the photons that cross the reporting planes and they poipulate the lower end of the momentum spectrium.

The bottom line is that change should not have a significant impact on physics but it will still be good to understand it.
